CHANGELOG.md

```diff
@@ -4,9 +4,27 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ## StreamChat
+### ✅ Added
+- [Threads v2] Add `ChatThreadListController` to fetch current user threads [#3229](https://github.com/GetStream/stream-chat-swift/pull/3229)
```

Comment:
I think that in one it is useful to be explicit and mention every new view etc... It is easier for the customer to indentify the new views. It also helps understand that it is in fact a big feature. It could be overlooked if it is just 1 entry point in the changelog. At least we have done this in the past, but let me know what do you think

Reply:
Not sure, for polls we just did one entry. Imo, if you're interested in the feature, you would look into the docs. With so many threads entries, you might miss some other important additions/fixes. 
What do the others think?

Reply:
Usually customers are lazy to open PRs and even Documentation sometimes 😄 IMO I feel like the explicit changelog helps quite a lot. Customers should be required to open PRs to check details of features, that should be just for us.

Anyways, I think we should add a link the full documentation in the changelog 👍

Reply:
If the list is quite short it is nice to see which functions were added so that it is easy to open the function in Xcode and read more about it. For larger changes, it makes sense to have a documentation link instead.

In this particular case, for me, it is OK to list all of them but maybe we should group them.
- [Threads v2] Add something something
-- Add `ChatMessageController.markThreadRead()` and `ChatMessageController.markThreadUnread()`
-- Add `ChatMessageController.updateThread()` for

Reply:
@laevandus @martinmitrevski I've updated the Changelog, let me know what you think